DNS/WobblyGuide.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "name": "wobbly.encode.moe.",
+        "type": "A",
+        "value": "185.199.111.153",
+        "proxied": true
+    }
```

Comment:
Other files (like https://github.com/Irrational-Encoding-Wizardry/infra/blob/main/DNS/Guide.json) have 4 A entries (and I remember these instructions generally being the same for all gh pages sites).

Reply:
I wasn't sure what to do about that or how to pick the values, so I figured I'd leave it to cid to explain to me.

Reply:
It's a way of load balancing since any client will pick one of the 4 IPs randomly.

Reply:
Should I commit, or are you going to be updating your PR?